src/main.js

```diff
@@ -638,30 +651,75 @@ window.LinkedinToResumeJson = (() => {
                         });
                     });
                 }
+                return true;
+            }
+        } catch (e) {
+            console.warn(e);
+            console.log('Error parsing using internal API (Voyager) - Contact Info');
+        }
+        return false;
+    }
+
+    LinkedinToResumeJson.prototype.parseViaInternalApiBasicAboutMe = async function parseViaInternalApiBasicAboutMe() {
+        try {
+            const basicAboutMe = await this.voyagerFetch(_voyagerEndpoints.basicAboutMe);
+            if (basicAboutMe && typeof basicAboutMe.data === 'object') {
+                if (Array.isArray(basicAboutMe.included) && basicAboutMe.included.length > 0) {
+                    const data = basicAboutMe.included[0];
+                    _outputJson.basics.name = `${data.firstName} ${data.LastName}`;
+                    // Note - LI labels this as "occupation", but it is basically the callout that shows up in search results and is in the header of the profile
+                    _outputJson.basics.label = data.occupation;
+                    _outputJson.basics.picture = data.picture.rootUrl + data.picture.artifacts[data.picture.artifacts.length - 1].fileIdentifyingUrlPathSegment;
+                }
+                return true;
+            }
+        } catch (e) {
+            console.warn(e);
+            console.log('Error parsing using internal API (Voyager) - Basic About Me');
+        }
+        return false;
+    }
+
+    LinkedinToResumeJson.prototype.parseViaInternalApiAdvancedAboutMe = async function parseViaInternalApiAdvancedAboutMe() {
+        try {
+            const advancedAboutMe = await this.voyagerFetch(_voyagerEndpoints.advancedAboutMe);
+            if (advancedAboutMe && typeof advancedAboutMe.data === 'object') {
+                const { data } = advancedAboutMe;
+                _outputJson.basics.name = `${data.firstName} ${data.lastName}`;
+                _outputJson.basics.label = data.headline;
+                _outputJson.basics.summary = data.summary;
+                return true;
+            }
+        } catch (e) {
+            console.warn(e);
+            console.log('Error parsing using internal API (Voyager) - AdvancedAboutMe');
+        }
+        return false;
+    };
+
+    // Outsourced all own parsing to own functions
+    LinkedinToResumeJson.prototype.parseViaInternalApi = async function parseViaInternalApi() {
+        try {
+            let apiSuccessCount = 0;
+            let fullProfileEndpointSuccess = false;
 
+            fullProfileEndpointSuccess = await this.parseViaInternalApiFullProfile();
+            if(fullProfileEndpointSuccess){
+                apiSuccessCount++;
+            }
+            if((await this.parseViaInternalApiFullSkills())){
```

Comment:
The use of double parenthesis here is not necessary, and goes against `prettier`'s rules.